rtb/verbali/esterni/VE_2024-12-04_ver1.0.0.typ

```diff
@@ -0,0 +1,69 @@
+#import "/templates/template.typ": *
+
+#show: content => verbale(
+  data: "04 dicembre 2024",
+  responsabile: "Davide Picello",
+  destinatari: (
+    "Prof. Tullio Vardanega",
+    "Prof. Riccardo Cardin",
+    "Sync Lab S.r.l.",
+  ),
+  uso: "Esterno",
+  redattori: ("Andrea Perozzo",),
+  titolo: "SAL intermedio secondo sprint",
+  verificatori: ("Riccardo Milan",),
+  riassunto: [Durante l'incontro sono state pianificate attività chiave, tra cui la creazione del _database_, l'integrazione con _Grafana_ e l'avanzamento del progetto.],
+  versioni: (
+    "1.0.0",
+    [05/12/2024],
+    "Andrea Perozzo",
+    "Riccardo Milan",
+    [- Stesura del verbale],
+  ),
+  content: content,
+  firmaRichiesta: true,
+)
+
+
+#infoRiunione(
+  luogo: "Google Meet",
+  data: "20/11/2024",
+  ora: "16:00",
+  durata: "40 minuti",
+  partecipantiEsterni: ("Fabio Pallaro", "Andrea Dorigo"),
+)
+
+= Sintesi dell'incontro
+Durante l'incontro si è discusso di diversi aspetti per l'avanzamento del progetto. È stata pianificata la creazione del _#rifGlossario("database")_, che verrà popolato con dati falsi e strutturato in tabelle che saranno molto probabilmente dedicate agli utenti (per la profilazione), ai punti di interesse e alle posizioni. Le posizioni pensiamo di organizzarle come serie temporali, collegate all'utente e, potenzialmente, agli annunci nella posizione specifica. Si è poi deciso che andremo a configurare un'immagine _#rifGlossario("Docker Compose")_ per semplificare la gestione del progetto. Un altro punto importante sarà andare ad integrare il _database_ con _#rifGlossario("Grafana")_, che servirà a creare una _dashboard_ per visualizzare in tempo reale le posizioni degli utenti.
+
+Per quanto riguarda i casi d'uso, continueremo a migliorarli e, se necessario, ne verranno scritti di nuovi. Parallelamente, verranno proseguite le seguenti attività di redazione:
+- _#rifGlossario("Sprint")_ 2 nel piano di progetto
+- Piano di qualifica
+- Norme di progetto.
+
+= Risposte alle domande
+Alcuni dubbi sono stati chiariti durante l'incontro. Per quanto riguarda la struttura del _database_, si è concordato di suddividerlo in tabelle separate per rendere più semplice l'analisi e l'integrazione con _Grafana_. La _dashboard_, infatti, si dovrà connettere direttamente al _database_ per mostrare in tempo reale i dati relativi alle posizioni degli utenti.
```

Comment:
- [ ] più che strutturare il database con tabelle separate (che è quasi scontato), il dubbio (che per altro non è ancora stato chiarito mi sembra) è se fare 2 database distinti per trattare le profilazioni e le posizioni gpx, oppure se mettere tutto in un unico db
- [ ] aggiungerei anche qui il discorso delle api di langchain che non funzionano, qui puoi chiedere a @klamerja più dettagli tecnici che ci ha lavorato (mi sembra il problema fosse che il processor usato è molto recente e ci sia poca documentazione
- [ ] aggiungerei il fatto che hanno dato feedback positivo rispetto a come stiamo lavorando su kafka e nifi